### PR TITLE
[zh] sync Misc Batch 1

### PR DESCRIPTION
**Pull request by CaoDonghui123**
part of #26178
sync :
 content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md
 content/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

### CODE REVIEW

File: content/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md [hunk 1]
```diff
@@ -519,21 +519,33 @@ CRDs converted from `apiextensions.k8s.io/v1beta1` to
...
-应该将 CustomResourceDefinition 自身的 `spec.preserveUnknownFields` 设置为
-`true`。
+对于使用 `apiextensions.k8s.io/v1beta1` 且将 `spec.preserveUnknownFields` 设置为 `true`
+创建的旧CustomResourceDefinition对象，以下条件也适用：
+
+* 裁剪未启用。
+* 可以存储任意数据。
+
+为了与`apiextensions.k8s.iov1`兼容，将你的自定义资源定义更新为：
+1. 使用OpenAPI 模式。
```
Comment (tengqm) [inline content/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md:546]:
```suggestion
1. 使用结构化的 OpenAPI 模式。
```

File: content/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md [hunk 2]
```diff
@@ -519,21 +519,33 @@ CRDs converted from `apiextensions.k8s.io/v1beta1` to
...
-要实现最佳的兼容性，你应该更新定制资源以满足某 OpenAPI 模式定义，并且你
-应该将 CustomResourceDefinition 自身的 `spec.preserveUnknownFields` 设置为
-`true`。
+对于使用 `apiextensions.k8s.io/v1beta1` 且将 `spec.preserveUnknownFields` 设置为 `true`
+创建的旧CustomResourceDefinition对象，以下条件也适用：
+
+* 裁剪未启用。
+* 可以存储任意数据。
+
+为了与`apiextensions.k8s.iov1`兼容，将你的自定义资源定义更新为：
```
Comment (tengqm) [inline content/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md:545]:
```suggestion
为了与 `apiextensions.k8s.io/v1` 兼容，将你的自定义资源定义更新为：
```

File: content/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md [hunk 3]
```diff
@@ -519,21 +519,33 @@ CRDs converted from `apiextensions.k8s.io/v1beta1` to
...
 从 `apiextensions.k8s.io/v1beta1` 转换到 `apiextensions.k8s.io/v1` 的 CRD
 可能没有结构化的模式定义，因此其 `spec.preserveUnknownFields` 可能为 `true`。
 
-对于迁移而来的 CustomResourceDefinition，如果其 `spec.preserveUnknownFields`
-被设置为 `true`，则 Kubernetes _不会_ 执行剪裁操作，你可以存储任意数据。
-要实现最佳的兼容性，你应该更新定制资源以满足某 OpenAPI 模式定义，并且你
-应该将 CustomResourceDefinition 自身的 `spec.preserveUnknownFields` 设置为
-`true`。
+对于使用 `apiextensions.k8s.io/v1beta1` 且将 `spec.preserveUnknownFields` 设置为 `true`
+创建的旧CustomResourceDefinition对象，以下条件也适用：
```
Comment (tengqm) [inline content/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md:540]:
```suggestion
创建的旧 CustomResourceDefinition 对象，有以下表现：
```

File: content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md [hunk 4]
```diff
@@ -55,18 +72,26 @@ by providing the following flags to the kube-apiserver:
...
 
-<!--
-1. Create an egress configuration file such as `admin/konnectivity/egress-selector-configuration.yaml`.
-1. Set the `--egress-selector-config-file` flag of the API Server to the path of
-your API Server egress configuration file.
--->
-{{< codenew file="admin/konnectivity/egress-selector-configuration.yaml" >}}
-2. 创建一个出口配置文件比如 `admin/konnectivity/egress-selector-configuration.yaml`。
-3. 将 API 服务器的 `--egress-selector-config-file` 参数设置为你的 API 服务器的
+1. 创建一个出口配置文件比如 `admin/konnectivity/egress-selector-configuration.yaml`。
```
Comment (tengqm) [inline content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md:75]:
```suggestion
1. 创建一个出站流量配置文件，比如 `admin/konnectivity/egress-selector-configuration.yaml`。
```

File: content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md [hunk 5]
```diff
@@ -55,18 +72,26 @@ by providing the following flags to the kube-apiserver:
...
+1. 将 API 服务器的 `--egress-selector-config-file` 参数设置为你的 API 服务器的
    离站流量配置文件路径。
+1. 如果您使用UDS连接，则将卷配置添加到kube-apiserver：
```
Comment (tengqm) [inline content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md:78]:
```suggestion
1. 如果你在使用 UDS 连接，须将卷配置添加到 kube-apiserver：
```

File: content/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md (hunk 1 above)
Comment (CaoDonghui123) [reply content/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md]: Thank you for your suggestions

File: content/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md [hunk 6]
```diff
@@ -519,21 +519,33 @@ CRDs converted from `apiextensions.k8s.io/v1beta1` to
...
-应该将 CustomResourceDefinition 自身的 `spec.preserveUnknownFields` 设置为
-`true`。
+对于使用 `apiextensions.k8s.io/v1beta1` 且将 `spec.preserveUnknownFields` 设置为 `true`
+创建的旧 CustomResourceDefinition 对象，有以下表现：
+
+* 裁剪未启用。
+* 可以存储任意数据。
+
+为了与 `apiextensions.k8s.iov1` 兼容，将你的自定义资源定义更新为：
+1. 使用结构化打 OpenAPI 模式。
```
Comment (tengqm) [inline content/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md:546]: 空行，拼写错：

```suggestion

1. 使用结构化的 OpenAPI 模式。
```

File: content/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md [hunk 7]
```diff
@@ -543,7 +543,7 @@ resource definitions to:
 * 可以存储任意数据。
 
 为了与 `apiextensions.k8s.iov1` 兼容，将你的自定义资源定义更新为：
-1. 使用结构化打 OpenAPI 模式。
+1. 使用结构化的 OpenAPI 模式。
```
Comment (tengqm) [inline content/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md:546]: 此行前面加空格。

另，将三个 commits suash成1个。

Comment (CaoDonghui123) [reply content/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md:547]: 好的

**Review by ydFu — APPROVED** [review]

**Review by SataQiu — COMMENTED** [review]
Thanks @CaoDonghui123 
/lgtm

**Pull request merged**